packages/devtools_app/web/index.html

```diff
@@ -147,6 +147,9 @@
             <div class="masthead-item action-button active" id="send-feedback-button" title="Send Feedback">
                 <span class="octicon octicon-comment"></span>
             </div>
+            <div class="masthead-item action-button active" id="try-flutter-web-devtools" title="Try DevTools on Flutter Web">
```

Comment:
this tooltip makes it sound like it might be telling you to use devtools for Flutter Web app apps.
How about
"Try alpha version of DevTools written with Flutter Web"
Or similar meaning but shorter.

Reply:
changed to "Try alpha version of DevTools written in Flutter!"